scrapy/core/downloader/handlers/http11.py

```diff
@@ -167,7 +169,7 @@ def processProxyResponse(self, rcvd_bytes):
                 extra = {'status': int(respm.group('status')),
                          'reason': respm.group('reason').strip()}
             else:
-                extra = rcvd_bytes[:32]
+                extra = rcvd_bytes[:_truncatedLength]
```

Comment:
I was confused as to why tests were passing, because you are using `_truncatedLength` instead of `self._truncatedLength` here.

Then I saw your comment about being confused yourself, and I think you mean that you don’t understand why the coverage complains. It turns out our current tests do not test this line, that’s why code coverage is incomplete. It was not covered before either. And new code should be covered by tests.

In this specific case, the code would have failed, which is why tests are so important. Please, have a look at the existing tests, and see if you can create a new test that covers this line. Otherwise, please let me know, maybe I can add one myself.

Reply:
I made the corrections for other changes you told me and went through how the tests work. It would be great if you can add a test for this particular case or give a clear idea about how to go through it.